src/v/cluster/controller_backend.cc

```diff
@@ -683,10 +684,47 @@ controller_backend::force_replica_set_update(
         co_return ss::stop_iteration::yes;
     }
 
+    auto enhanced_force_reconfiguration_enabled = _features.local().is_active(
+      features::feature::enhanced_force_reconfiguration);
+    std::vector<raft::vnode> voters;
+    std::vector<raft::vnode> learners;
+    voters.reserve(new_replicas.size());
+    learners.reserve(new_replicas.size());
```

Comment:
nit: not really needed, as in all cases the containers are moved into these vars.